docs/blog/newsletter-december-2020.rst

```diff
@@ -0,0 +1,112 @@
+
+.. post:: December 03, 2020
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – December 2020
+#########################################
+
+Hey everyone - it's Beth bringing you the final WTD newsletter of the year.
+
+I hope you're all holding up okay. I haven't got any grand thoughts for you on how to sum up this year, or how we'll handle what's next. But I do want to thank all of you for being part of this community - for your conversations, your contributions, your insights and support and wit and sarcasm. They're what Write the Docs is built on. So thanks, for making our Slack and the three conferences into a source of support and connection - in a year when we've needed it more than ever.
```

Comment:
Slight preference for "how to sum up this year for you" over "for you on how to sum up this year" but YMMV

Reply:
The rest is <3 <3 <3

Reply:
I agree, I like it a lot. After the year we've all had I think a little extra <3 is welcome.

Reply:
Changed to Sam's suggestion, and thank you!